torchbox/torchbox.com
# Home page design review

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1422206676)

### Description of Changes Made

Lots of design review feedback on the homepage. I tried to keep these to separate commits but there have ended up being too many, so I'll just summarise the changes below.

- Don't output empty grid if breadcrumb is not present
- Adjust padding on all buttons
- Changes to event block display (and rename the sass file)
- Changes to promo block display
- Make client and meta info more consistent across the board
- A few mobile changes - images no longer full-width, listing font-size and spacing, home page icon is smaller, event and partners block layout have changed, hide featured case study results icon
- Further changes to the featured case study to try and get it to work better at different viewports. I realised the new results ring wasn't cropped which was affecting the positioning so I've tweaked the viewbox
- Feedback on the tabbed paragraph block
- Alignment, title size and spacing on home page

### How to Test

Most of these components are on the home page, so you could make sure you have a full home page (pulling staging data I think will help here) - or view in the pattern library. I'm not adding screenshots as there would be a lot but shout if any of the changes don't make sense.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
